content/installation/java/Java-basic-installation.md

```diff
@@ -0,0 +1,20 @@
+<!--
+title: "Java Agent Installation"
+description: "Overview of the Java Agent installation"
+tags: "installation Java agent overview installation"
+-->
+
+To install the Java agent, complete the following steps:
+
+* Log in to the Contrast interface.
+* Click the button in the top navigation bar to **Add Agent**.
+* Select the **Java** agent in the dropdown menu, and click the button to **Download Agent**. You may need to specify proxy authentication information required by your network before downloading the agent.
+
+> **Note:** The **Java** agent can be used with all compatible versions of Java, and upgrades automatically upon restart if a newer version is available. The **Java 1.5** agent doesn't upgrade automatically upon restart.
```

Comment:
Should we specify that if the customer is using java 5, they must use the java 1.5 agent?

Reply:
clarified.